Userland/Libraries/LibGUI/Widget.h

```diff
@@ -299,6 +301,9 @@ class Widget : public Core::Object {
 
     bool has_pending_drop() const;
 
+    // In order for others to be able to call this, it needs to be public.
+    virtual bool load_from_gml_ast(NonnullRefPtr<GUI::GML::Node> ast, RefPtr<Core::Object> (*unregistered_child_handler)(const String&));
```

Comment:
Who outside of a Widget hierarchy is calling load_from_gml_ast on a Widget? It looks like it could be protected, like load_from_json() was, as load_from_gml() takes a StringView and a function pointer and is the API that apps are actually calling to load their GML hierarchy into existence.

Unless the intent is to allow the GML playground to hold onto an AST instead of Strings in the future?

Reply:
Thing is, I of course thought this as well (and that's why the comment is there). But for whatever reason, ScrollableWidgetContainer can't call the function on arbitrary widgets if it's not public. I know, weird.